README.md

```diff
@@ -70,53 +304,184 @@ Make sure you have Postgres, Redis and Vault properly installed & configured. Do
+--header 'Authorization: Basic dXNlci1pc3N1ZXI6cGFzc3dvcmQtaXNzdWVy';
 
-If you are setting up [locally](#setup-for-docker-only) with Docker, you will need to set up the following variables in their respective `.env` files.
+# Expected Output:
+#   {"body":{"credentials":[{"description":"https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/kyc-v3.json-ld#KYCAgeCredential","id":"b1eab5be-dea3-11ed-8f7d-0242ac1e0005"}],"url":"http://localhost:3001/v1/agent"},"from":"did:polygonid:polygon:mumbai:2qPdb2hNczpXhkTDXfrNmmt9fGMzfDHewUnqGLahYE","id":"c01b66f1-d10b-4591-9dd6-8a94db1c112a","thid":"c01b66f1-d10b-4591-9dd6-8a94db1c112a","to":"did:polygonid:polygon:mumbai:2qEsg1AeTohAq6Euc3hBaDapfLVfQiWS7DUfvutYEq","typ":"application/iden3comm-plain-json","type":"https://iden3-communication.io/credentials/1.0/offer"}
+```
+
+Take this JSON data, copy, and paste into [https://qr.io](https://qr.io).
+
+!["QR.io"](/docs/qrio.png)
 
-In `.env-api`:
+With the Polygon ID app, open it up and scan the QR code.
 
-- `ISSUER_API_UI_AUTH_USER`
-- `ISSUER_API_UI_AUTH_PASSWORD`
-- `ISSUER_API_UI_ISSUER_DID` - obtained when following the steps in [creating the issuer DID](#creating-the-issuer-did).
-- `ISSUER_ETHEREUM_URL` - this is the URL of the issuer's DApp.
-- `ISSUER_API_UI_ISSUER_LOGO` - optional (placeholder used if left blank). A valid URL to a minimum 40x40 pixel PNG, JPEG or SVG of the issuer's logo.
+!["Polygon ID App Adding Claim/Credential"](/docs/polygonid-app-claim.png)
 
-In `.env-issuer`:
+### Verifying Claim
 
-- `ISSUER_API_AUTH_USER`
-- `ISSUER_API_AUTH_PASSWORD`
-- `ISSUER_KEY_STORE_TOKEN` - obtained when following the steps in [adding Ethereum private key to the Vault](#adding-ethereum-private-key-to-the-vault).
+ℹ️ **NOTE:** The goal is to build your own type of claims and ways to verify claims, but this is an example of how things could work.
 
-If you are running the UI, in `.env-ui`:
+A quick way to validate this KYCAge Claim is to use [https://verifier-demo.polygonid.me/](https://verifier-demo.polygonid.me/).
 
-- `ISSUER_UI_AUTH_USERNAME`
-- `ISSUER_UI_AUTH_PASSWORD`
+!["Verifier Selecting KYCAgeCredential"](/docs/verifier-kycagecredential.png)
 
-### Adding Ethereum private key to the Vault
+!["Verifier Verification Prompt"](/docs/verifier-verification.png)
 
-This is required for signing on-chain transactions. In a basic use-case this can be retrieved from an Ethereum wallet that can connect to Polygon Mumbai Testnet.
+!["Polygon ID App Generating Proof"](/docs/polygonid-app-proof.png)
 
-Follow these steps:
+!["Verifier Proof Verified"](/docs/verifier-success-verified.png)
 
-1. Copy your Ethereum private key, pasting it into `<private_key>` in the next step.
-2. Run `make private_key=<private_key> add-private-key`
-3. Run `make add-vault-token`
+---
+## How To Run The Server
 
-### Creating the issuer DID
+### Running the admin server backend (api) for evaluation purposes with docker-composer
```

Comment:
This should day "Running the API for the UI". Why is this here though? The UI is a crucial part of being able to submit schemas and issue credentials directly, via links to QR codes and managing issuer state and largely supersede the manual `curl` commands or using the Issuer API. These instructions are also duplicated in [line 29](https://github.com/0xPolygonID/sh-id-platform/pull/327/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R29).

Reply:
Yep, I think I missed this in the re-arranging. Agree UI would be key. I need to test it a bit further, because I couldn't get it working as it is, but I'm guessing it's still being worked on.